content/accessibility/checklist.mdx

```diff
@@ -0,0 +1,148 @@
+    - [ ]  Screen reader labels
+    - [ ]  Landmarks (if applicable)
+- [ ]  Does the text meet color contrast (4.5:1 or 3:1 ratio depending on size)?
+- [ ]  Do graphic objects meet color contrast (3:1 ratio)?
+- [ ]  With color removed, does the design still make sense?
+- [ ]  If the user increases the text size, is the page still usable?
+- [ ]  When the screen reduces in size (down to 320px wide), is everything usable without scrolling horizontally? (There are a few exceptions to this rule: check  [1.4.10 - Reflow](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html) for details)
+
+When designing a new component or control that isn't made up of one single piece of [standard semantic markup](https://developer.mozilla.org/en-US/docs/Web/HTML/Element), check out the Accessibility Portal's [custom control design checklist](https://www.a11yportal.com/guidelines/fundamentals/checklists.html#custom-control-checklist-ux).
+
+## Visual considerations
+
+### Color
+<!-- TODO: insert photo of our colors with pass/fail-->
+<!-- TODO: the following link doesn't exist.
+For information about how GitHub meets color contrast guidelines, read the accessibility section under the color guidelines.-->
+**Note:** Disabled controls do not need to hit any specific contrast levels. 
+
+**Official WCAG rules**
+- [1.4.3 - Contrast (Minimum)(Level AA)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)
+- [1.4.11 - Non-text Contrast (Level AA)](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html)
+- [1.4.1 - Use of color (Level AA)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html)
+
+### Typography
+#### Headings
+<!-- TODO: insert photo of proper headings, labeled -->
+Paragraphs and sections should have approriate headings — use the semantic structure for headings whenever possible (`h1`- `h6`). This makes it easier for visual scanning and screen readers to get a map of the page's content.
+
+**Note:** Properly labeled headings is one of the most important things a designer can label for developer handoff. [68% of 1207 surveyed users](https://webaim.org/projects/screenreadersurvey8/) responded that they are first likely to navigate through the headings on the page when trying to find information on a length web page.
+
+#### Length of text
+Though a level AAA guideline, try to hit the following typography standards for better ease of reading:
+- Width of text should not exceed 80 characters or glyphs 
+- Don't justify text
+- Allow for text magnification up to 200% without requiring horizontal scrolling.
+
+Read Primer's [typography](https://primer.style/design/foundations/typography) guidelines for more detail about using type.
+
+#### Links
+Navigating from a list of all the links on a given web page is very common for assistive technology users. Make sure that the link text itself is meaningful and unique. There should be as few duplicated references as possible to prevent confusion. 
```

Comment:
@ichelsea I tried to read the first sentence a few times but having trouble understanding it. Can you rewrite it to be easier to follow?

Reply:
How is, "Assistive technology users will frequently browse a list of links on a web page for quicker navigation." ?

Reply:
Like this?

```suggestion
Assistive technology users will frequently browse a list of links for easy of navigation. For this reason, make sure that the link text is meaningful and unique. There should be as few duplicated references as possible to prevent confusion.
```

Reply:
Yeah! I edited "easy of navigation" to "easy navigation". Should it say "ease of navigation"? Also, side note, super weird that I can edit your comments??

Reply:
Sounds good, yeah I guess it's a GitHub thing 😄